tests/phpunit/unit/api/actions/beansModifyActionPriority.php

```diff
@@ -0,0 +1,109 @@
+<?php
+/**
+ * Tests for beans_modify_action_priority()
+ *
+ * @package Beans\Framework\Tests\Unit\API\Actions
+ *
+ * @since   1.5.0
+ */
+
+namespace Beans\Framework\Tests\Unit\API\Actions;
+
+use Beans\Framework\Tests\Unit\API\Actions\Includes\Actions_Test_Case;
+
+require_once __DIR__ . '/includes/class-actions-test-case.php';
+
+/**
+ * Class Tests_BeansModifyActionPriority
+ *
+ * @package Beans\Framework\Tests\Unit\API\Actions
+ * @group   unit-tests
+ * @group   api
+ */
+class Tests_BeansModifyActionPriority extends Actions_Test_Case {
+
+	/**
+	 * Setup test fixture.
+	 */
+	protected function setUp() {
+		parent::setUp();
+
+		require_once BEANS_TESTS_LIB_DIR . 'api/actions/functions.php';
+		require_once BEANS_TESTS_LIB_DIR . 'api/utilities/functions.php';
+	}
+
+	/**
+	 * Reset the test fixture.
+	 */
+	protected function tearDown() {
+		parent::tearDown();
+
+		global $_beans_registered_actions;
+		$_beans_registered_actions = array(
+			'added'    => array(),
+			'modified' => array(),
+			'removed'  => array(),
+			'replaced' => array(),
+		);
+	}
+
+	/**
+	 * Test beans_modify_action_priority() should return false when the ID is not registered.
+	 */
+	public function test_should_return_false_when_id_not_registered() {
+		$ids = array(
+			'foo'   => null,
+			'bar'   => 0,
+			'baz'   => 10,
+			'beans' => '20',
+		);
+		foreach ( $ids as $id => $priority ) {
+			$this->assertFalse( beans_modify_action_priority( $id, $priority ) );
+		}
+	}
+
+	/**
+	 * Test beans_modify_action_callback() should return false when null is new priority.
+	 */
+	public function test_should_return_false_when_priority_is_non_integer() {
+		$ids = array(
```

Comment:
Comment doesn't match function.
Maybe `Test beans_modify_action_callback() should return false when new priority is not an integer.` ?

( `Test beans_modify_action_arguments() should return false when new args is a non-integer value.` is used in `tests/phpunit/integration/api/actions/beansModifyActionArguments.php` )